<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fontEditor</class>
 <widget class="QDialog" name="fontEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Font editor</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="buttonsLayout">
     <item>
      <widget class="QPushButton" name="load">
       <property name="text">
        <string>Load...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="save">
       <property name="text">
        <string>Export...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="nameLayout">
     <item>
      <widget class="QLabel" name="l_fname">
       <property name="text">
        <string>Font name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="fontSelect">
       <item>
        <property name="text">
         <string>&lt;New...&gt;</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="rename">
       <property name="text">
        <string>Rename...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="remove">
       <property name="text">
        <string>Remove...</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="sizeLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="l_height">
         <property name="text">
          <string>Character height</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fHeight">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="l_width">
         <property name="text">
          <string>Character width</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="fWidth">
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="editLayout">
     <item>
      <layout class="QVBoxLayout" name="labelLayout">
       <item>
        <widget class="QLabel" name="curpos">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Cursor Position</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="l_avChars">
           <property name="text">
            <string>Characters:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="charList">
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QPushButton" name="removeChar">
           <property name="text">
            <string>Remove this char</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="nChar">
           <property name="text">
            <string>New character:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="charName">
           <property name="maximumSize">
            <size>
             <width>40</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="maxLength">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="newChar">
           <property name="text">
            <string>New</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fillStd">
           <property name="text">
            <string>Insert standard characters...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="importFromImage">
           <property name="text">
            <string>Import char from image...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
